test/test.js

```diff
@@ -14,13 +14,14 @@ var benchmarkLibs = fs.readdirSync('benchmarks').filter(function (script) {
 describe('main test suite', function () {
   it('test sourcemaps', function () {
     var res = optimizeJs('var baz = function () { console.log("foo") }()', {
-      sourceMap: true
+      file: 'optimized.js',
+      source: 'original.js',
+      sourceMap: true,
+      includeContent: true
```

Comment:
So it looks like this includes three new options - `file`, `source`, and `includeContent`. Could you update the README please? :)